LDKNodeMonday/View Model/Profile/NetworkSettingsViewModel.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  NetworkSettingsViewModel.swift
+//  LDKNodeMonday
+//
+//  Created by Daniel Nordh on 20/01/2025.
+//
+
+import LDKNode
+import SwiftUI
+
+class NetworkSettingsViewModel: ObservableObject {
+    private let keyClient: KeyClient
+    @Binding var walletClient: WalletClient
+    @State private var showRestartAlert = false
+    @Published var selectedNetwork: Network = .signet {
+        didSet {
+            //walletClient.appState != .onboarding ? try keyClient.saveNetwork(selectedNetwork.description) : nil
```

Comment:
I think we can remove this line if not needed anymore

Reply:
Removed (in fact the whole file is removed)